it/repositories/ListRepositorySpec.scala

```diff
@@ -35,7 +36,7 @@ class ListRepositorySpec
   import ListRepositorySpec._
 
   override def beforeAll(): Unit = {
-    database.flatMap(_.collection[JSONCollection](ListCollection.collectionName).drop(failIfNotFound = false)).futureValue
```

Comment:
Man, I tried everything to get this working and it looks like we just have to drop everything. Great stuff @amaalali (I'd commented trait out in order of progressing it!)

Reply:
I did too, I was trying to keep the drop operation as light as possible but 🤷